### PR TITLE
Delivery Instructions

### PR DESCRIPTION
**Pull request by metas-rc**
https://github.com/metasfresh/metasfresh/issues/14215

### CODE REVIEW

File: backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5671690_sys_gh14215_Delivery_Instruction_Notification_message.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+-- Value: Event_DeliveryInstructionGenerated
+-- 2023-01-13T13:30:32.339Z
+INSERT INTO AD_Message (AD_Client_ID,AD_Message_ID,AD_Org_ID,Created,CreatedBy,EntityType,IsActive,MsgText,MsgType,Updated,UpdatedBy,Value) VALUES (0,545223,0,TO_TIMESTAMP('2023-01-13 15:30:31','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','Delivery Instruction Created: {0}.','I',TO_TIMESTAMP('2023-01-13 15:30:31','YYYY-MM-DD HH24:MI:SS'),100,'Event_DeliveryInstructionGenerated')
```
Comment (metas-ts) [inline backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5671690_sys_gh14215_Delivery_Instruction_Notification_message.sql:3]: please prefix thise AD_Messages with `de.metas.deliveryplanning.`

File: backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5672080_sys_gh14215_M_Delivery_Planning_Delivery_Instructions_V.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+       di.createdby,
+       sp.M_ShippingPackage_ID as M_Delivery_Planning_Delivery_Instructions_V_ID,
+       di.updated,
+       di.updatedby,
+       di.isactive,
+       di.ad_org_id,
+       di.ad_client_id
+FROM M_ShipperTransportation di
+         JOIN M_ShippingPackage sp
+              ON di.m_shippertransportation_id = sp.m_shippertransportation_id
```
Comment (metas-ts) [inline backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5672080_sys_gh14215_M_Delivery_Planning_Delivery_Instructions_V.sql:33]: please make sure to have indexes on
- M_ShippingPackage.m_shippertransportation_id
- M_Delivery_Planning.releaseno

**Review by metas-ts — CHANGES_REQUESTED** [review]

File: backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5671690_sys_gh14215_Delivery_Instruction_Notification_message.sql (hunk 1 above)
Comment (metas-rc) [reply backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5671690_sys_gh14215_Delivery_Instruction_Notification_message.sql:3]: done in separate script

File: backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5672080_sys_gh14215_M_Delivery_Planning_Delivery_Instructions_V.sql (hunk 2 above)
Comment (metas-rc) [reply backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5672080_sys_gh14215_M_Delivery_Planning_Delivery_Instructions_V.sql:33]: * M_ShippingPackage.m_shippertransportation_id already exists

**Review by metas-ts — APPROVED** [review]

**Pull request merged**